numpy/lib/arraysetops.py

```diff
@@ -160,6 +168,9 @@ def unique(ar, return_index=False, return_inverse=False,
         The number of times each of the unique values comes up in the
         original array. Only provided if `return_counts` is True.
         .. versionadded:: 1.9.0
+    unique_flags : ndarray, optional
+        The flags of the first occurrences of the unique values in the
+        original array. Only provided if `return_flags` is True.
```

Comment:
`mask` would be a much better name than `flags` here, I think. (and in `return_flags`, obviously)

I see you chose it as it matched the internal `flag` variable, but I don't think that was a good name either.